src/org/omegat/Bundle.properties

```diff
@@ -3020,14 +3020,34 @@ PREFS_SECURE_STORAGE_RESET_MASTER_PASSWORD_MESSAGE=Are you sure you want to rese
 # PREFERENCES - PLUGINS
 PREFS_TITLE_PLUGINS=Plugins
 
+# org.omegat.gui.dialogs.PluginInstallerDialog
+PREFS_PLUGINS_DETAILS_explanation=Show plugin details
+PREFS_PLUGINS_COL_STAT=Status
+PREFS_PLUGINS_COL_CATEGORY=Category
+PREFS_PLUGINS_COL_NAME=Name
+PREFS_PLUGINS_COL_VERSION=Version
+PREFS_PLUGINS_UPGRADE=Upgrade
+PREFS_PLUGINS_INSTALL=Install
+
+PLUGIN_STATUS_BUNDLED=Bundled
+PLUGIN_STATUS_UPGRADABLE=Upgradable
```

Comment:
Following from the above:
- PLUGIN_STATUS_UPDATABLE=Update available

Updatable is better for the key name, but "update available" is more natural for users.